### DISCUSSION TITLE
Let's add a bit more horizontal space

### DISCUSSION THREAD

**Discussion by igorwojda** [community/community#66327] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

We could improve these test cutoffs because there is so much free space on the screen
<img width="1468" alt="image" src="https://github.com/community/community/assets/530122/146ed0f5-dfea-4016-976f-879f9c3619f0">

**Comment by mansourshebli** [comment]
Agree
